src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -740,6 +722,7 @@ void NetworkOPsImp::processClusterTimer ()
     if (!update)
     {
         JLOG(m_journal.debug()) << "Too soon to send cluster update";
+        setClusterTimer ();
```

Comment:
Yes.  If we entered this branch then the clusterTimer would never be set again.  This was a bug.  Hopefully we never entered this branch...